pkg/virt-controller/watch/vm.go

```diff
@@ -1419,6 +1419,15 @@ func (c *VMController) stopVMI(vm *virtv1.VirtualMachine, vmi *virtv1.VirtualMac
 	}
 
 	// stop it
+	// if for some reason the VM has been requested to be deleted, we want to use the
+	// deletion grace period specified to the VM as the TerminationGracePeriodSeconds
+	// for the VMI.
+	if vm.DeletionTimestamp != nil {
+		err = c.patchVMITerminationGracePeriod(vm.GetDeletionGracePeriodSeconds(), vmi)
+		if err != nil {
+			log.Log.Object(vmi).Errorf("unable to patch vmi termination grace period: %v", err)
```

Comment:
Is it correct to ignore the error here?

Reply:
I think you can also simply return the err, in this case it should return a syncErr which will cause a new sync cycle.

Reply:
I've updated it, let me know what you think